### PR TITLE
testapps and gpu packagegroups

### PR DESCRIPTION
**Pull request by nsdrude**

### CODE REVIEW

File: recipes-fsl/packagegroups/packagegroup-fsl-tools-testapps.bb [hunk 1]
```diff
@@ -15,6 +15,15 @@ SOC_TOOLS_TEST_imx = "imx-test"
 SOC_TOOLS_TEST_imxgpu  = "imx-test imx-gpu-viv-demos"
 SOC_TOOLS_TEST_qoriq = "ceetm optee-test-qoriq"
 
+SOC_TOOLS_IMX_TESTAPPS = " \
+    vlan \
+    cryptodev-module \
+    cryptodev-tests \
```
Comment (otavio) [inline recipes-fsl/packagegroups/packagegroup-fsl-tools-testapps.bb:21]: Any reason to add those?

File: recipes-fsl/packagegroups/packagegroup-fsl-tools-testapps.bb [hunk 2]
```diff
@@ -37,6 +46,24 @@ RDEPENDS_${PN} = " \
...
+    can-utils \
+    cpufrequtils \
+    nano \
+    ntpdate \
+    minicom \
+    coreutils \
+    mmc-utils \
+    udev-extraconf \
+    e2fsprogs-resize2fs \
+    openssl-bin \
```
Comment (otavio) [inline recipes-fsl/packagegroups/packagegroup-fsl-tools-testapps.bb:63]: This does not make sense. We are not trying to provide a full image here but the bare minimum to validate the devices.

This belongs to a custom image but not here.

**Review by otavio — CHANGES_REQUESTED** [review]

**Comment by otavio** [comment]
Also, this needs to be sent on top of master. We cannot apply this to dunfell without very good reasoning.

**Comment by nsdrude** [comment]
@otavio to simplify, I will split this into two different pull requests. I'll start by creating a pull request to master for:

- https://github.com/Freescale/meta-freescale-distro/pull/53/commits/f2ca0c7e2b725efcbe6e94b64e1af9b37fa2da2e
- https://github.com/Freescale/meta-freescale-distro/pull/53/commits/1dca8bf28176def176fd18c2b6518d764995b4da

Do you have any feedback for these patches before I submit an updated pull request?

Thanks,
Nate

**Comment by otavio** [comment]
You can close this PR and we handle all this on the specific PRs :-)

**Pull request closed without merging**